lib/worker.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+if (!process.features.experimental_workers) {
+  throw new Error('Experimental workers are disabled');
+}
+
+const util = require('util');
+const assert = require('assert');
+const EventEmitter = require('events');
+const WorkerContextBinding = process.binding('WorkerContext');
```

Comment:
it's not documented, but it's pretty standard to use `worker_context` syntax for binding exports.